src/pybind/mgr/dashboard/frontend/src/app/shared/api/auth.service.ts

```diff
@@ -13,10 +14,10 @@ export class AuthService {
 
   login(credentials: Credentials) {
     return this.http
-      .post('api/auth', credentials)
+      .post('api/auth', { username: credentials.username, password: credentials.password })
```

Comment:
Since we now only have username and password on the Credentials class, maybe we can keep the old code and pass the variable as the body of POST?